### ISSUE TITLE
How to create a Pull Request

### ISSUE THREAD

**Issue by timmwille** [ASKnet-Open-Training/Github-Guidance#9]
![Pull-Request-2020-10-18 20-04](https://user-images.githubusercontent.com/8081055/101546629-5dbf3380-39a9-11eb-869b-983972a0194f.gif)

**Comment by PyGamma** [comment]
This is great work Timm, Additional Instruction below to add on to your working.
1. First is to commit or otherwise the changes you have done in you branch.
2. Once you push the changes to your repo, the Compare & pull request button will appear in GitHub as shown below 
![compare-and-pull-request-button](https://user-images.githubusercontent.com/38918594/104121571-61e7c380-5350-11eb-89f0-a4de35e91540.png)
3. Click it and you'll be taken to this screen:
![open-a-pull-request_crop](https://user-images.githubusercontent.com/38918594/104121598-99567000-5350-11eb-8efa-f34b0c28b6b6.png)
4. Open a pull request by clicking the Create pull request button. This allows the repo's maintainers to review your contribution. From here, they can merge it if it is good, or they may ask you to make some changes.

In summary, if you want to contribute to a project, the simplest way is to:

Find a project you want to contribute to
Fork it
Clone it to your local system if you want to work on it locally
Make a new branch
Make your changes
Push it back to your repo
Click the Compare & pull request button
Click Create pull request to open a new pull request
If the reviewers ask for changes, repeat steps 5 and 6 to add more commits to your pull request.

**Comment by bmen** [comment]
The page in the guide is already created, but the content still needs to be added.
https://asknet-open-training.github.io/Github-Guidance/pages/github-repo/pull-request-create/

The content can be easily added by clicking "Edit this page on GitHub" at the very bottom of the page. This will take you directly to the source.
![readme-edit-this-page](https://user-images.githubusercontent.com/389294/104121836-ad46a580-5341-11eb-8159-167ee6980000.png)
